.changeset/afraid-cups-deny.md

```diff
@@ -0,0 +1,5 @@
+---
+"rdf-validate-shacl": minor
+---
+
+Added support for `sh:shapesGraph` and `owl:imports`. BREAKING: `.validate` and `.validateNode` methods are now asynchronous
```

Comment:
I don't see `sh:shapesGraph` support yet. Am I missing something?

Reply:
Yes of of course. Good catch. I forgot to update the changeset since we discussed whet to implement